--- PDCurses-3.9.orig/curses.h	2019-09-04 22:08:02.000000000 +0200
+++ PDCurses-3.9/curses.h	2020-09-23 11:09:29.955553000 +0200
@@ -187,6 +187,7 @@
 #define PDC_MOUSE_WHEEL_RIGHT   0x0100
 
 #define A_BUTTON_CHANGED        (Mouse_status.changes & 7)
+#undef  MOUSE_MOVED
 #define MOUSE_MOVED             (Mouse_status.changes & PDC_MOUSE_MOVED)
 #define MOUSE_POS_REPORT        (Mouse_status.changes & PDC_MOUSE_POSITION)
 #define BUTTON_CHANGED(x)       (Mouse_status.changes & (1 << ((x) - 1)))
--- PDCurses-3.9.orig/wincon/Makefile	2019-09-04 22:08:02.000000000 +0200
+++ PDCurses-3.9/wincon/Makefile	2020-09-23 08:14:00.000000000 +0200
@@ -26,11 +26,11 @@
 
 PDCURSES_WIN_H	= $(osdir)/pdcwin.h
 
-CC		= gcc
-AR		= ar
-STRIP		= strip
-LINK		= gcc
-WINDRES		= windres
+CC		= $(TARGET)gcc
+AR		= $(TARGET)ar
+STRIP		= $(TARGET)strip
+LINK		= $(TARGET)gcc
+WINDRES		= $(TARGET)windres
 
 ifeq ($(DEBUG),Y)
 	CFLAGS  = -g -Wall -DPDCDEBUG
--- PDCurses-3.9.orig/wincon/pdckbd.c	2019-09-04 22:08:02.000000000 +0200
+++ PDCurses-3.9/wincon/pdckbd.c	2024-11-20 13:11:57.329144000 +0100
@@ -210,7 +210,37 @@
    {0,          0,         0x27,        ALT_FQUOTE, 0   }, /* 222 */
    {0,          0,         0,           0,          0   }, /* 223 */
    {0,          0,         0,           0,          0   }, /* 224 */
-   {0,          0,         0,           0,          0   }  /* 225 */
+   {0,          0,         0,           0,          0   }, /* 225 */
+   {0,          0,         0,           0,          0   }, /* 226 */
+   {0,          0,         0,           0,          0   }, /* 227 */
+   {0,          0,         0,           0,          0   }, /* 228 */
+   {0,          0,         0,           0,          0   }, /* 229 */
+   {0,          0,         0,           0,          0   }, /* 230 */
+   {0,          0,         0,           0,          0   }, /* 231 */
+   {0,          0,         0,           0,          0   }, /* 232 */
+   {0,          0,         0,           0,          0   }, /* 233 */
+   {0,          0,         0,           0,          0   }, /* 234 */
+   {0,          0,         0,           0,          0   }, /* 235 */
+   {0,          0,         0,           0,          0   }, /* 236 */
+   {0,          0,         0,           0,          0   }, /* 237 */
+   {0,          0,         0,           0,          0   }, /* 238 */
+   {0,          0,         0,           0,          0   }, /* 239 */
+   {0,          0,         0,           0,          0   }, /* 240 */
+   {0,          0,         0,           0,          0   }, /* 241 */
+   {0,          0,         0,           0,          0   }, /* 242 */
+   {0,          0,         0,           0,          0   }, /* 243 */
+   {0,          0,         0,           0,          0   }, /* 244 */
+   {0,          0,         0,           0,          0   }, /* 245 */
+   {0,          0,         0,           0,          0   }, /* 246 */
+   {0,          0,         0,           0,          0   }, /* 247 */
+   {0,          0,         0,           0,          0   }, /* 248 */
+   {0,          0,         0,           0,          0   }, /* 249 */
+   {0,          0,         0,           0,          0   }, /* 250 */
+   {0,          0,         0,           0,          0   }, /* 251 */
+   {0,          0,         0,           0,          0   }, /* 252 */
+   {0,          0,         0,           0,          0   }, /* 253 */
+   {0,          0,         0,           0,          0   }, /* 254 */
+   {0,          0,         0,           0,          0   }  /* 255 */
 };
 
 static KPTAB ext_kptab[] =
